src/Enchantments.cpp

```diff
@@ -175,32 +175,32 @@ int cEnchantments::StringToEnchantmentID(const AString & a_EnchantmentName)
 		const char * m_Name;
 	} EnchantmentNames[] =
 	{
-		{ enchProtection,           "Protection"},
-		{ enchFireProtection,       "FireProtection"},
-		{ enchFeatherFalling,       "FeatherFalling"},
-		{ enchBlastProtection,      "BlastProtection"},
-		{ enchProjectileProtection, "ProjectileProtection"},
-		{ enchRespiration,          "Respiration"},
-		{ enchAquaAffinity,         "AquaAffinity"},
-		{ enchThorns,               "Thorns"},
-		{ enchDepthStrider,         "DepthStrider"},
-		{ enchSharpness,            "Sharpness"},
-		{ enchSmite,                "Smite"},
-		{ enchBaneOfArthropods,     "BaneOfArthropods"},
-		{ enchKnockback,            "Knockback"},
-		{ enchFireAspect,           "FireAspect"},
-		{ enchLooting,              "Looting"},
-		{ enchEfficiency,           "Efficiency"},
-		{ enchSilkTouch,            "SilkTouch"},
-		{ enchUnbreaking,           "Unbreaking"},
-		{ enchFortune,              "Fortune"},
-		{ enchPower,                "Power"},
-		{ enchPunch,                "Punch"},
-		{ enchFlame,                "Flame"},
-		{ enchInfinity,             "Infinity"},
-		{ enchLuckOfTheSea,         "LuckOfTheSea"},
-		{ enchLure,                 "Lure"},
-	} ;
+		{ enchProtection, "Protection" },
+		{ enchFireProtection, "FireProtection" },
+		{ enchFeatherFalling, "FeatherFalling" },
+		{ enchBlastProtection, "BlastProtection" },
+		{ enchProjectileProtection, "ProjectileProtection" },
+		{ enchRespiration, "Respiration" },
+		{ enchAquaAffinity, "AquaAffinity" },
+		{ enchThorns, "Thorns" },
+		{ enchDepthStrider, "DepthStrider" },
+		{ enchSharpness, "Sharpness" },
+		{ enchSmite, "Smite" },
+		{ enchBaneOfArthropods, "BaneOfArthropods" },
+		{ enchKnockback, "Knockback" },
+		{ enchFireAspect, "FireAspect" },
+		{ enchLooting, "Looting" },
+		{ enchEfficiency, "Efficiency" },
+		{ enchSilkTouch, "SilkTouch" },
+		{ enchUnbreaking, "Unbreaking" },
+		{ enchFortune, "Fortune" },
+		{ enchPower, "Power" },
+		{ enchPunch, "Punch" },
+		{ enchFlame, "Flame" },
+		{ enchInfinity, "Infinity" },
+		{ enchLuckOfTheSea, "LuckOfTheSea" },
+		{ enchLure, "Lure" },
```

Comment:
Not sure how those got changed, as I didn't touch it.  But I have added spaces back.  Weird that a merge with upstream didn't put it back.